src/lib/config.js

```diff
@@ -53,6 +53,9 @@ function parseFeaturesConfig () {
 }
 
 function parseKeysConfig () {
+  const isProduction = process.env.NODE_ENV === 'production'
+  const mustSignNotifications = Config.castBool(
+    Config.getEnv(envPrefix, 'NOTIFICATION_SIGN'), isProduction)
```

Comment:
We already have a `CONNECTOR_NOTIFICATION_VERIFY`config option though. We should be consistent.